pkg/server/server.go

```diff
@@ -90,6 +94,66 @@ func (s *Server) SetRules(w http.ResponseWriter, r *http.Request, tenant string)
 	_, _ = w.Write([]byte("successfully updated rules file"))
 }
 
+func (s *Server) ListAllRules(w http.ResponseWriter, r *http.Request) {
+	logger := log.With(s.logger, "handler", "listAllRules")
+
+	//nolint:exhaustivestruct
```

Comment:
What is this linter? :thinking: Do we need it allowed?

Reply:
It throws an error if we initialize a struct without initializing all of its fields.

I am of the same opinion as you on this. Almost everywhere this linter complained, I had to put a `//nolint` comment. I'd be in favor for disabling this one.

Reply:
I see, I never heard of this linter before, I think we should disable it, since having uninitialized struct can be fairly common. But let's not keep this PR hanging longer, I'm resolving this.

Reply:
Added https://github.com/observatorium/api/issues/212 since I think same applies to API, but we could adjust all lint config in one go